pkg/crc/api/client/client.go

```diff
@@ -215,14 +215,20 @@ func (c *Client) SetPullSecret(data string) error {
 	return nil
 }
 
-func (c *Client) sendGetRequest(url string) ([]byte, error) {
-	res, err := c.client.Get(fmt.Sprintf("%s%s", c.base, url))
+func (c *Client) sendGetRequest(url string, data io.Reader) ([]byte, error) {
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s%s", c.base, url), data)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/json")
+
+	res, err := c.client.Do(req)
 	if err != nil {
 		return nil, err
 	}
 	defer res.Body.Close()
 
-	if res.StatusCode != http.StatusOK {
+	if res.StatusCode != http.StatusOK && res.StatusCode != http.StatusCreated {
```

Comment:
`http.StatusCreated` is unexpected on a `get` request

Reply:
`Created` mostly comes from a `PUT`: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT